src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -30,10 +30,10 @@
   <ItemGroup>
     <PackageReference Include="Apache.Arrow" Version="2.0.0" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
-    <PackageReference Include="Microsoft.Data.Analysis" Version="0.4.0" />
+    <PackageReference Include="Microsoft.Data.Analysis" Version="0.18.0" />
```

Comment:
Should we update to the newest version, `0.19.0` ?

Reply:
I see that @pgovind had verified for no breaking changes with 0.18.0 so thought we should stick with that till we get a confirmation that 0.19.0 doesn't break anything.  Looks like `dotnet-interactive` also currently supports 0.18.0 - https://github.com/dotnet/interactive/blob/main/src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.DotNet.Interactive.ExtensionLab.csproj#L27?